app/views/AttitudeBarView.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * @author Bingzheng Feng
+ * @module views/AttitudeBarView
+ * @requires models/Commands
+ * @requires models/TelemetryData
+ * @requires util/Template
+ * @requires electron
+ * @listens models/TelemetryData~TelemetryData:data_received
+ * @copyright Waterloo Aerial Robotics Group 2016
+ * @licence https://raw.githubusercontent.com/UWARG/WARG-Ground-Station/master/LICENSE
+ * @description Responsible for displaying and setting the aircraft's attitude rate (roll rate, pitch rate, yaw rate)
+ */
+
+var remote = require('electron').remote;
+var Template = require('../util/Template');
+var TelemetryData = remote.require('./app/models/TelemetryData');
+var Commands = remote.require('./app/models/Commands');
+
+module.exports = function (Marionette) {
+
+  return Marionette.ItemView.extend({
+    template: Template('AttitudeBarView'),
+    className: 'attitudeBarView',
+
+    ui: {
+      pitch_rate_rad: '.pitch.value-rad',
+      pitch_rate_deg: '.pitch.value-deg',
+      pitch_front: '.pitch.block.front',
+      pitch_input: '.input.pitch',
+      yaw: '.yaw-value',
```

Comment:
this is not used or updated. THe yaw value is not needed in this view, just the rates, so you can delete this as well as the corresponding html element.